app/src/main/java/com/example/zar/shopistantcontent/DetailsFragment.java

```diff
@@ -0,0 +1,84 @@
+package com.example.zar.shopistantcontent;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+
+import java.util.HashMap;
+
+/**
+ * Created by Zar on 4/10/2017.
+ */
+
+public class DetailsFragment extends Fragment {
+    private  int quantity=0;
+    private  String itemId;
+    private String aislePosition;
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view=inflater.inflate(R.layout.fragment_products,container,false);
```

Comment:
this is not the best practice to supply everything in onCreateView because if we need to use the same below code somewhere then we cannot re-use it, so not a good approach